locales/es/es.json

```diff
@@ -300,11 +301,11 @@
     "Kiribati": "Kiribati",
     "Korea": "Corea del Sur",
     "Korea, Democratic People's Republic of": "Corea del Norte",
-    "Korea, Republic of": "Korea, Republic of",
+    "Korea, Republic of": "Corea, República de",
```

Comment:
Republic of Korea is South Korea

Reply:
[https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L553](https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L553) is duplicated in `source/en.json`